wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreateSecureGuestLink/CreateSecureGuestLinkViewModel.swift

```diff
@@ -62,6 +72,31 @@ final class CreateSecureGuestLinkViewModel {
         return true
     }
 
+    func createSecuredGuestLinkIfValid(conversation: ZMConversation, passwordField: ValidatedTextField, confirmPasswordField: ValidatedTextField) {
+        guard validatePassword(for: passwordField, against: confirmPasswordField) else {
+            delegate?.viewModel(self, didFailToValidatePasswordWithReason: "Password validation failed.")
+            return
+        }
+
+        let password = passwordField.text ?? ""
+        UIPasteboard.general.string = password
```

Comment:
Why is it needed to write the password to the user's system wide pasteboard?

Reply:
We had this discussion 1:1, it's according to design.